### PR TITLE
PUBLISH: 5/12 Query builder link from logging ui

### PR DESCRIPTION
**Pull request by eknuth**
Hoping to get this published on 5/12.

Adds documentation for new link from logging ui to nrdb query builder. Adds "what's new" post and updates the main logging ui docs to add an additional section for this functionality.

### CODE REVIEW

File: src/content/docs/logs/ui-data/use-logs-ui.mdx [hunk 1]
```diff
@@ -145,10 +145,27 @@ To explore your logging data, follow this basic workflow.
...
+    id="query-builder"
+    title="7. Advanced log analysis with NRDB query builder."
+  >
+  <img
+    title="Advanced log analysis with NRDB query builder"
+    alt="Screenshot of link to NRDB query builder"
+    src={logsNrqlButton}
+  />
+
+  Go beyond the capabilities of the streamlined querying interface of the logging UI by writing NRQL queries on log events directly, using the [NRDB query builder](/docs/query-your-data/explore-query-data/query-builder/introduction-query-builder/). Click the "NRQL" button to the right of the search bar to go directly to this powerful NRQL interface, without losing the columns and lucene filtering you have already selected.
```
Comment (rkbodenner) [inline src/content/docs/logs/ui-data/use-logs-ui.mdx:160]:
```suggestion
  Go beyond the capabilities of the streamlined querying interface of the Logs UI by writing NRQL queries on log events directly, using the [NRDB query builder](/docs/query-your-data/explore-query-data/query-builder/introduction-query-builder/). Click the "NRQL" button to the right of the Logs search bar to go directly to this powerful NRQL interface, without losing the columns and Lucene filtering you have already selected.
```

**Comment by eknuth** [comment]
Thanks, Ralph! This is good to publish.

**Review by aalfano7 — APPROVED** [review]
Approved with committed changes (to md file)

**Review by aalfano7 — DISMISSED** [review]

**Review by akristen — APPROVED** [review]
I made some light copy changes and will merge when Gatsby builds!

**Pull request merged**